src/core.js

```diff
@@ -231,7 +233,36 @@ Core.prototype.init = function() {
     this._sceneFirstPass.add(this._meshFirstPass);
     this._sceneSecondPass.add(this._meshSecondPass);
     //this._sceneSecondPass.add(this._axes);
-    
+      
+////FramesPerSecond
```

Comment:
This is a very weird commenting style. Why here 4 slashes and later 2 slashes, not to mention the indetation level for these comments doesn't align with the function below. If someone new is to look at this code, they will get a headache first.